src/material/slide-toggle/slide-toggle.ts

```diff
@@ -193,7 +192,13 @@ export class MatSlideToggle extends _MatSlideToggleMixinBase implements OnDestro
     this._focusMonitor
       .monitor(this._elementRef, true)
       .subscribe(focusOrigin => {
-        if (!focusOrigin) {
+        // Only forward focus manually when it was received programmatically or through the
+        // keyboard. We should not do this for mouse/touch focus for two reasons:
+        // 1. It can prevent clicks from landing in Chrome (see #18269).
+        // 2. They're already handled by the wrapping `label` element.
+        if (focusOrigin === 'keyboard' || focusOrigin === 'program') {
+          this._inputElement.nativeElement.focus();
```

Comment:
Related to that, did we consider forwarding the focus origin here? (through `focusVia`)?

Reply:
I don't mind changing it if you feel strongly about it. I didn't do it, because there's nothing reacting to different focus origins on the input anyway so it seemed redundant.

Reply:
Yeah, it was mostly for sanity. I don't feel strongly about it. Let's just leave it as is.